src/lints/struct_with_no_pub_fields_changed.ron

```diff
@@ -0,0 +1,76 @@
+SemverQuery(
+    id: "struct_with_no_pub_fields_changed",
+    human_readable_name: "public API struct with no public fields is no longer a struct",
+    description: "A struct without pub fields was converted into an enum or union, breaking pattern matching.",
```

Comment:
should the description rather be 
```suggestion
    description: "A struct without pub fields, but with non-pub fields was converted into an enum or union, breaking pattern matching.",
```

Reply:
Probably not necessary, I think. So long as the lint always fires when it should, I don't think all the edge cases have to be enumerated in the user-facing text. That sounds probably more confusing than helpful.